build.gradle

```diff
@@ -230,6 +230,7 @@ project(':ambry-commons') {
         compile ("org.apache.helix:helix-core:$helixVersion") {
             exclude group: "log4j", module: "log4j"
         }
+        compile "mysql:mysql-connector-java:8.0.21"
```

Comment:
Could you add $mysqlVersion and use it here.  Also, can we remove it from account project?

Reply:
updated